GenderPayGap.Database.Models/Return.cs

```diff
@@ -49,7 +48,6 @@ public Return()
         public bool EHRCResponse { get; set; }
 
         public virtual Organisation Organisation { get; set; }
-        public virtual ICollection<Organisation> Organisations { get; set; }

```

Comment:
But each `Return` also has an `Organisations` collection (which is always unused)
This only existed because it mirrors the link from Organisation.LatestReturn to Return (since a Return could, in theory, be linked to by multiple Organisations via their LatestReturn properties)

Reply:
Thanks for the note